pkg/apis/v1beta1/aksnodeclass.go

```diff
@@ -159,6 +159,11 @@ type KubeletConfiguration struct {
 // AKSNodeClass is the Schema for the AKSNodeClass API
 // +kubebuilder:object:root=true
 // +kubebuilder:resource:path=aksnodeclasses,scope=Cluster,categories={karpenter,nap},shortName={aksnc,aksncs}
+// +kubebuilder:printcolumn:name="Ready",type=string,JSONPath=".status.conditions[?(@.type=='Ready')].status"
```

Comment:
I would skip SubnetID even in wide, anything long makes the output unwieldy. (You can still request it explicitly if needed)

Reply:
fair